client/hwlib/src/utils.rs

```diff
@@ -0,0 +1,73 @@
+/* Copyright 2024 Canonical Ltd.
```

Comment:
Generally speaking, I like to avoid modules named 'util' as they risk becoming a misc bag of unrelated things and there's almost always a better name they can be given

Reply:
I've updated it with `helpers` name and added `test_utils` module there for tests
Let me know if you want to see something cleverer here